Resources/Prototypes/GameRules/cargo_gifts.yml

```diff
@@ -1,3 +1,42 @@
+# Add your new event to this RandomRule if you want it to happen via the basic/ramping schedulers.
+- type: entity
+  parent: BaseGameRule
+  id: RandomCargoGiftsRule
+  components:
+  - type: RandomRule
+    selectableGameRules:
+    # Make the probability sum to 1 or I will remove your fingernails with toothpicks.
```

Comment:
you dont need to make it sum to one. this uses weights for a reason.

Reply:
Need is relative, balancing 85 gamerules with random weights sounds like cbt to me, but if you guys would rather keep track of that than the clear and concise "it happens 15% of the time" its no skin off my back.

Reply:
All the probabilities are the same you may as well remove the yaml lines and make it significantly easier to maintain.